docs/src/persistence.md

```diff
@@ -1,14 +1,10 @@
 # Reading and writing Graphs
 
-Graphs may be written to I/O streams and files using the `savegraph` function and read with the `loadgraph` function. The default graph format is a proprietary compressed *LightGraphs.jl* format `lg`. Other formats are available via the *GraphPersistence* package.
+## Saving using *LightGraphs.jl* `lg` format.
 
-```@autodocs
-Modules = [LightGraphs]
-Pages   = [ "persistence/common.jl"]
-Private = false
-```
+Graphs may be written to I/O streams and files using the `savegraph` function and read with the `loadgraph` function. The default graph format is a proprietary compressed *LightGraphs.jl* format `lg`.
```

Comment:
Note that other graph implementations add methods to `savegraph` and `loadgraph`, so these functions are not limited to SimpleGraphs.

Reply:
Proprietary is not really correct in this usage. The format is FOSS licensed just like the software that defines it. We mean `custom` or `bespoke` not `commercial` or `copy-righted`

Definition of proprietary from Merriam Webster:

> 2: something that is used, produced, or marketed under exclusive legal right of the inventor or maker; specifically :a drug (such as a patent medicine) that is protected by secrecy, patent, or copyright against free competition as to name, product, composition, or process of manufacture

Reply:
+1 to "bespoke". Though - @jpfairbanks - it'd be great to have an mmaped file format :)

Reply:
hahaha -- that was copy paste off the old docs! Updating to bespoke now.